### ISSUE TITLE
support falsy values for value in mapping?

### ISSUE THREAD

**Issue by nfcampos** [heroku/react-refetch#83]
Hi,

The condition at https://github.com/heroku/react-refetch/blob/master/src/components/connect.js#L51 causes a request like
```js
connect(props => {
  return {
    blahFetch: {
      url: '/some/url'
      then: value => {
        if (value.some.key)
          return '/some/other/url'
        else
          return {value: null}
      }
    }
  }
})(MyComponent)
```
to be invalid. I understand the need to validate there is a `url` or `value`, however, how should I do a request like this? (ie. a request in which you poll a first url, you use the response of that to check whether something exists, if it exists you do the request you really want, otherwise that first request doesn't need a value)

Nuno

**Comment by ryanbrainard** [comment]
Ugh, we should probably be using `mapping.hasOwnProperty('value')` instead. I can try to get a fix out for you.

**Comment by ryanbrainard** [comment]
Fix is at https://github.com/heroku/react-refetch/pull/84

**Comment by nfcampos** [comment]
Looks great, thanks!

**Comment by ryanbrainard** [comment]
Released in [v0.7.1](https://github.com/heroku/react-refetch/releases/tag/v0.7.1)
